test/api/test_custom_op.cpp

```diff
@@ -0,0 +1,23 @@
+#include <migraphx/migraphx.h>
+#include <migraphx/migraphx.hpp>
+#include "test.hpp"
+
+struct my_custom_op final : migraphx::experimental_custom_op_base
```

Comment:
should we rename this to `simple_custom_op`? Then similarly in L16 rename `cop` to `simple_op`?